NEWS.rst

```diff
@@ -106,6 +106,9 @@ unambiguously guessed from the atom name, in accordance with PDB structures.
 Bio.PDB entities now have a ``center_of_mass()`` method that calculates either
 centers of gravity or geometry.
 
+Bio.PDB.PDBParser and Bio.PDB.MMCIFParser now also read `pdb.gz` and `cif.gz`
+files.
```

Comment:
I believe that that was already the case?

Reply:
Yeah I forgot to take this out. I will change the news and add the remaining docs when we have all agreed that the pull request is ready.